### PR TITLE
Use of checkers on docker_cli_inspect_experimental_test.go

### PR DESCRIPTION
**Pull request by liaoqingwei**
Signed-off-by: liaoqingwei liaoqingwei@huawei.com

### CODE REVIEW

**Comment by HuKeping** [comment]
Thanks @liaoqingwei this change seems good.

**Comment by vdemeester** [comment]
@liaoqingwei I think there is a few comments that could be kept (without the variables in it), just to add some context in case of test failure.
Other than that, looks good :wink:

File: integration-cli/docker_cli_inspect_experimental_test.go [hunk 1]
```diff
@@ -12,34 +13,22 @@ func (s *DockerSuite) TestInspectNamedMountPoint(c *check.C) {
...
 
-	if m.Driver != "local" {
-		c.Fatalf("Expected driver local, was %s\n", m.Driver)
-	}
+	c.Assert(m.Driver, checker.Equals, "local")
 
-	if m.Source == "" {
-		c.Fatalf("Expected source to not be empty")
-	}
+	c.Assert(m.Source, checker.Equals, "")
```
Comment (HuKeping) [inline integration-cli/docker_cli_inspect_experimental_test.go]: check.Not(checher.Equals)

Comment (vdemeester) [inline integration-cli/docker_cli_inspect_experimental_test.go]: Oh, missed that one, thanks @HuKeping :wink:

Comment (liaoqingwei) [inline integration-cli/docker_cli_inspect_experimental_test.go]: Update，please review again，thanks @vdemeester  @HuKeping

**Comment by vdemeester** [comment]
LGTM 🐸

**Comment by estesp** [comment]
LGTM

**Pull request merged**